### DISCUSSION TITLE
Alias python keyword

### DISCUSSION THREAD

**Discussion by FraserLee** [xonsh/xonsh#5104] [General]
Hi. In zsh, I had aliases set-up so I could enter n dots to cd up n-1 directories. I'd love to be able to assign `aliases['...'] = 'cd ../..'`, but entering the string `...` in my prompt just returns the python object `Ellipsis`. 

Is there any way to get around this, or is the limitation fundamental?

**Comment by anki-code** [comment]
The simplest workaround is to use [xontrib-abbrevs](https://github.com/xonsh/xontrib-abbrevs):
```xsh
xontrib load abbrevs
abbrevs['...'] = 'cd ../..'
...
# cd ../..
```

**Reply by anki-code** [reply]
You can also bind hotkeys for this to reduce keystrokes entirely - [read cheatsheet](https://github.com/anki-code/xonsh-cheatsheet/blob/main/README.md#bind-hotkeys-in-prompt-toolkit-shell).
